scripts/utils/trading_strategy_helpers.js

```diff
@@ -75,6 +77,33 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     return orders != null
   }
 
+  /**
+   * Returns the symbol of the ERC20 token in input.
+   * Returns the velue from a list if present, otherwise executes a contract call.
+   *
+   * @param {string} method the name of the methods that (can be "symbol", "decimals", or "name")
```

Comment:
Would it be wise to use here an enum?

(maybe not worth the effort as the whole project is not typed)

Reply:
Never heard about enum, seems like it could be a good idea.

Reply:
(After discussing privately, we decided to merge without enums.)